tests/functional/curate/cram/rename/general.t

```diff
@@ -0,0 +1,66 @@
+Setup
+
+  $ export AUGUR="${AUGUR:-$TESTDIR/../../../../../bin/augur}"
+
+The tests here use NDJSON I/O and don't explicitly test TSV I/O as we rely
+on the general curate infrastructure to enforce that each row has the same
+fields. See <https://github.com/nextstrain/augur/issues/1510> for more
+
+  $ cat >records.ndjson <<~~
+  > {"accession": "record_1", "country": "country_1"}
+  > {"accession": "record_2", "country": "country_2"}
+  > ~~
+
+Test that running without (rename-specific) arguments doesn't modify data
+
+  $ $AUGUR curate rename < <(cat records.ndjson) > out1.ndjson
+
+  $ diff records.ndjson out1.ndjson
```

Comment:
Huh, this stood out to me because it would seem weird to run the command without any arguments. I think that `--field-map` should be required.

Reply:
Updated in force-push (by making it a required arg)